src/screens/PaymentDetailsScreen.tsx

```
@@ -5,25 +1,98 @@
+            className="input"
+            type="text"
+            placeholder="Payment title"
+            value={paymentTitle}
+            onChange={(e) => setPaymentTitle(e.target.value)}
+          />
+        </div>
+      </div>
+      <h2 className="subtitle">Shares</h2>
+      {team.map((member) => (
+        <div key={member.name} className="field">
+          <label className="label">{member.name}</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)